mlir/test/Dialect/LLVMIR/global.mlir

```diff
@@ -205,56 +205,26 @@ llvm.mlir.global internal @g(43 : i64) : i64 {
 
 // -----
 
-llvm.mlir.global internal @g(32 : i64) {addr_space = 3: i32} : i64
-func.func @mismatch_addr_space_implicit_global() {
-  // expected-error @+1 {{pointer address space must match address space of the referenced global}}
-  llvm.mlir.addressof @g : !llvm.ptr
-  llvm.return
-}
-
-// -----
-
-llvm.mlir.global internal @g(32 : i64) {addr_space = 3: i32} : i64
-
-func.func @mismatch_addr_space() {
```

Comment:
It looks like you accidentally remove some tests here? Or did they go somewhere else?